src/elements/Badge/Badge.tsx

```diff
@@ -80,15 +101,16 @@ export const Badge: FC<BadgeProps> = ({ children, status, icon, style = BadgeSty
                 {status && (
                     <span
                         data-test-id="badge-status"
-                        className={`tw-w-2 tw-h-2 tw-rounded-full ${statusClasses[status]}`}
+                        className={merge(["tw-w-2 tw-h-2 tw-rounded-full", statusClasses[status]])}
+                        style={status.indexOf("#") === 0 ? { backgroundColor: status } : {}}
```

Comment:
Also allow other color inputs not just hex codes

Reply:
I validate against isBadgeStatus now